pulse_actions/handlers/treeherder_resultset.py

```diff
@@ -14,18 +14,22 @@ def on_resultset_action_event(data, message, dry_run):
     # Pulse gives us resultset_id, we need to get revision from it.
     resultset_id = data["resultset_id"]
     treeherder_client = TreeherderClient()
+    # We do not handle 'cancel_all' action right now, so skip it.
+    if action == "cancel_all":
+        message.ack()
+        return
```

Comment:
We don't need to publish messages via MessageHandler() for 'cancel_all' action since we do not process it right now.